apps/src/studio/MusicController.js

```diff
@@ -188,3 +201,22 @@ MusicController.prototype.whenMusicStopped_ = function (musicName) {
     this.nowPlaying_ = null;
   }
 };
+
+/**
+ * @param {string} name
+ * @returns {MusicTrack|undefined}
+ * @private
+ */
+MusicController.prototype.getTrackByName_ = function (name) {
+  return _.find(this.trackList_, function (track) {
+    return track.name === name;
+  });
+};
+
+/**
+ * @returns {MusicTrack|undefined}
+ * @private
+ */
+MusicController.prototype.getRandomTrack_ = function () {
+  return this.trackList_[Math.floor(Math.random(this.trackList_.length))];
```

Comment:
Right now:
1. You list all the songs you might want to use in the skin with their metadata.  This does not cause them to be loaded.
2. You can list any number of songs on the level.  All songs listed on the level _will_ be loaded so they are ready to use in that level.  Right now we are only listing one song per hoc2015 level, so we are only loading one song per level.
3. As currently written, we randomly select one song from the _level_ list (not the skin list) to play when the instructions show.  Since we only list one song per level right now, we guarantee which song will play.